neo.UnitTests/UT_Consensus.cs

```diff
@@ -47,10 +47,10 @@ public void ConsensusService_Primary_Sends_PrepareRequest_After_OnStart()
             int timeIndex = 0;
             var timeValues = new[] {
               //new DateTime(1968, 06, 01, 0, 0, 15, DateTimeKind.Utc), // For tests here
-              new DateTime(1968, 06, 01, 0, 0, 1, DateTimeKind.Utc),  // For receiving block
-              new DateTime(1968, 06, 01, 0, 0, (int) Blockchain.SecondsPerBlock, DateTimeKind.Utc), // For Initialize
-              new DateTime(1968, 06, 01, 0, 0, 15, DateTimeKind.Utc), // unused
-              new DateTime(1968, 06, 01, 0, 0, 15, DateTimeKind.Utc)  // unused
+              new DateTime(1980, 06, 01, 0, 0, 1, 001, DateTimeKind.Utc),  // For receiving block
+              new DateTime(1980, 06, 01, 0, 0, (int) Blockchain.MillisecondsPerBlock / 1000, 100, DateTimeKind.Utc), // For Initialize
+              new DateTime(1980, 06, 01, 0, 0, 15, 001, DateTimeKind.Utc), // unused
+              new DateTime(1980, 06, 01, 0, 0, 15, 001, DateTimeKind.Utc)  // unused
```

Comment:
Why do we need these unused lines?

Reply:
I am not sure, it was probably something when we are doing the first draft of the UT for consensus.
But better to handle this in another PR if you want to modify something related.

Reply:
Agreed